layers/meta-tegrademo/recipes-bsp/u-boot/u-boot-tegra/0015-t210-mender-FIT-image-bootcmd-support.patch

```diff
@@ -0,0 +1,107 @@
+index 1a8fa81151..849e32b091 100644
+--- a/include/env_default.h
++++ b/include/env_default.h
+@@ -35,8 +35,8 @@ const uchar default_environment[] = {
+ #ifdef	CONFIG_USE_BOOTARGS
+ 	"bootargs="	CONFIG_BOOTARGS			"\0"
+ #endif
+-#ifdef	CONFIG_MENDER_BOOTCOMMAND
+-	"bootcmd="	CONFIG_MENDER_BOOTCOMMAND	"\0"
++#ifdef	CONFIG_MENDER_FITFDT_BOOTCOMMAND
++	"bootcmd="	CONFIG_MENDER_FITFDT_BOOTCOMMAND			"\0"
+ #endif
+ #ifdef	CONFIG_RAMBOOTCOMMAND
+ 	"ramboot="	CONFIG_RAMBOOTCOMMAND		"\0"
+@@ -86,6 +86,9 @@ const uchar default_environment[] = {
+ #ifdef	CONFIG_LOADADDR
+ 	"loadaddr="	__stringify(CONFIG_LOADADDR)	"\0"
+ #endif
++#ifdef CONFIG_FITFDT_LOADADDR
++	"fitfdt_loadaddr="	__stringify(CONFIG_FITFDT_LOADADDR)	"\0"
++#endif
+ #if defined(CONFIG_PCI_BOOTDELAY) && (CONFIG_PCI_BOOTDELAY > 0)
+ 	"pcidelay="	__stringify(CONFIG_PCI_BOOTDELAY)"\0"
+ #endif
+diff --git a/include/env_mender.h b/include/env_mender.h
+index fbed122807..4ac9657343 100644
+--- a/include/env_mender.h
++++ b/include/env_mender.h
+@@ -154,6 +154,24 @@
+     "sysboot ${devtype} ${devnum}:${distro_bootpart_hex} any ${scriptaddr} ${prefix}extlinux/extlinux.conf; " \
+     "run mender_try_to_recover"
+ 
++#define CONFIG_MENDER_FITFDT_BOOTCOMMAND                                \
++    "run mender_setup; "                                                \
++    "setenv distro_bootpart ${mender_boot_part}; "                      \
++    "setenv distro_bootpart_hex ${mender_boot_part_hex}; "              \
++    "setenv devnum " __stringify(MENDER_UBOOT_STORAGE_DEVICE) "; "      \
++    "setenv devtype " __stringify(MENDER_UBOOT_STORAGE_INTERFACE) "; "  \
++    "setenv prefix /boot/; "                                            \
++    "setenv load_fit_image load ${devtype} ${devnum}:${distro_bootpart_hex} ${fitfdt_loadaddr} ${prefix}FitImage; "  \
++    MENDER_BOOTARGS                                                     \
++    "setenv bootargs ${bootargs} rw rootwait; "                         \
++    "if test \"${systemd_machine_id}\" != \"\"; then "                  \
++    "setenv bootargs ${bootargs} systemd.machine_id=${systemd_machine_id}; " \
++    "fi; "                                                              \
++    "setenv bootargs ${cbootargs} ${bootargs}; "                        \
++    "run load_fit_image; "                                              \
++    "bootm ${fitfdt_loadaddr}#conf@_ddot___ddot___ddot___ddot__nvidia_platform_t210_porg_kernel-dts_tegra210-p3448-0000-p3449-0000-a02.dtb; " \
++    "run mender_try_to_recover;"
++
+ #endif /* !MENDER_AUTO_PROBING */
+ 
+ #endif /* HEADER_ENV_MENDER_H */
+diff --git a/scripts/config_whitelist.txt b/scripts/config_whitelist.txt
+index 093e432efc..5bf5657a4d 100644
+--- a/scripts/config_whitelist.txt
++++ b/scripts/config_whitelist.txt
+@@ -508,6 +508,7 @@ CONFIG_FEROCEON_88FR131
+ CONFIG_FILE
+ CONFIG_FIRMWARE_OFFSET
+ CONFIG_FIRMWARE_SIZE
++CONFIG_FITFDT_LOADADDR
```

Comment:
See my previous question about why this is needed.

Reply:
I will try without and let you know.